modules/lang-painless/src/main/java/org/elasticsearch/painless/node/SSource.java

```diff
@@ -130,7 +130,8 @@ public int getMaxLoopCounter() {
     private final ScriptClassInfo scriptClassInfo;
     private final CompilerSettings settings;
     private final String name;
-    private final String source;
+    // TODO can this be removed all up the call chain?
+    // private final String source;
```

Comment:
@jdconrad I couldn't find any reference to this field, but maybe its here for debugging purposes or anything the like? In that case I would add a comment instead of removing. If this is really unused, maybe it can also be removed from the method signatures passing it down here. Could you take a look and comment?

Reply:
Will take a look!  Thanks for pointing this out.

Reply:
@cbuescher Looks like this is now extraneous.  It never got removed once source was set in the class above.  Please feel free to remove this as part of this PR or if you'd prefer I can remove it in a different PR.

Reply:
will do, thanks